### DISCUSSION TITLE
Are the docs up to date?

### DISCUSSION THREAD

**Discussion by dylanrhodius** [vercel/next.js#29389] [Help]
Are we sure the docs are up to date? 
I followed the steps to setup a Contenful blog using the steps on [this link](https://github.com/vercel/next.js/tree/canary/examples/cms-contentful).

When running the project though, I wasn't able to fetch anything from the cms (wrong access token).
![emptyBlog](https://user-images.githubusercontent.com/21088670/134788723-23832f8d-d9ef-4e54-8907-1feef509ea1a.png)
<img width="853" alt="wrongAccessToken" src="https://user-images.githubusercontent.com/21088670/134788746-cb3fc705-c097-49d7-8653-df735d32ee25.png">

After a lot of head scratching and investigating, turns out I had to switch the branch on Contentful to use `main` instead of `master`.
![mainvsMaster](https://user-images.githubusercontent.com/21088670/134788744-5f198817-b4ea-4fc6-95c9-b24ef7d5d68b.png)

I then ran into a problem where I needed to create a `next.config.js` file with the following code 
```
module.exports = {
  images: {
    domains: ['images.ctfassets.net'],
  },
}
```

So my question is: am I even using the right tutorial to setup a nextjs blog with a headless cms?

**Comment by ItsJamesMurray** [comment]
Did you setup the content models ([step 2](documentToReactComponents)) manually or using the setup?  When I did it manually I had the same issue, but deleting it all in Contentful and then running the setup script it solved that issue.

**Reply by dylanrhodius** [reply]
Interesting! I believe I did go the manual way first, but I then also used the script and it still wouldn't work for me..
